libs/api/domains/cms/src/lib/dto/getArticles.input.ts

```diff
@@ -24,4 +24,9 @@ export class GetArticlesInput {
   @Max(1000)
   @IsOptional()
   size?: number = 100
+
+  @Field({ nullable: true })
+  @IsString()
+  @IsOptional()
+  sort?: 'title' | 'popular' = 'title'
```

Comment:
I added the `@IsEnum()` so that GraphQL can return `sort must be a valid enum value` when an invalid value is used.

I tried creating an `enum sortDirection { title, popular }` but I'm not sure how to use it since `input.sort` is a string but `sortDirection.popular` is an integer.

Reply:
Something like this might do it.
``` ts
import { registerEnumType } from '@nestjs/graphql'

export enum SortDirection {
  DESC = 'desc',
  ASC = 'asc',
}

registerEnumType(ContentLanguage, { name: 'SortDirection' })
```

Then use like this:
``` ts
@Field(() => SortDirection, { nullable: true })
```

Reply:
Something like this might do it.
``` ts
import { registerEnumType } from '@nestjs/graphql'

export enum SortDirection {
  DESC = 'desc',
  ASC = 'asc',
}

registerEnumType(SortDirection, { name: 'SortDirection' })
```

Then use like this:
``` ts
@Field(() => SortDirection, { nullable: true })
```